scripts/cronjob.sh

```diff
@@ -1 +1,3 @@
-* 1 * * * /usr/bin/python /biblib/biblib/manage.py syncdb >> /tmp/biblib_delete_stale_users.log
\ No newline at end of file
+* 1 * * * /usr/bin/python /biblib/biblib/manage.py syncdb >> /tmp/biblib_delete_stale_users.log
+* 1 * * * /usr/bin/python /biblib/biblib/manage.py clean_versions_number >> /tmp/biblib_revision_deletion.log
+* 1 * * * /usr/bin/python /biblib/biblib/manage.py clean_versions_time >> /tmp/biblib_revision_deletion.log
```

Comment:
I don't think we actually use this cronjob setup anymore - it's all in Beehive. No harm in having it here too, but we'll have to add it to Beehive, once we're ready (for posterity, we agreed to check in again a couple months after deployment to see how big the versions table was getting, and how quickly, to fine tune the cleanup).

Reply:
That makes sense, I mainly put it here so that way we have the exact syntax recorded somewhere for if and when we need it.